### DISCUSSION TITLE
Are you encountering any problem with long CVE numbers?

### DISCUSSION THREAD

**Discussion by mertssmnoglu** [orhun/flawz#25] [Q&A]
I'm encountering with #24 on my laptop. I can not see the last char on the long CVE numbers.

Do you also encounter this problem like me? Or is everything ok with long CVEs?

**Comment by orhun** [comment]
Same here:

![image](https://github.com/orhun/flawz/assets/24392180/fe597c2c-94c9-46ff-a061-baa7ba989ea1)
